test/functional/services/remote/webdriver.ts

```diff
@@ -44,6 +44,7 @@ import { Browsers } from './browsers';
 const throttleOption: string = process.env.TEST_THROTTLE_NETWORK as string;
 const headlessBrowser: string = process.env.TEST_BROWSER_HEADLESS as string;
 const remoteDebug: string = process.env.TEST_REMOTE_DEBUG as string;
+const certValidation: string = process.env.NODE_TLS_REJECT_UNAUTHORIZED as string;
```

Comment:
Sorry, one more thing - can we keep the naming convention for the environment variable? 

Using something like `TEST_IGNORE_CERTIFICATE_ERRORS` would also align with being truthy when enabled, but open to other names.